new-pipeline/src/evo.js

```diff
@@ -809,10 +807,10 @@ function displayEvolution(target, lines, usePercentages, plotOptions) {
   $(`${target} .mg-x-axis .label`)
     .attr("text-decoration", "underline")
     .click(function () {
-      var newUseSubmissionDate = $("input[name=build-time-toggle]:checked")
-        .val() !== "0" ? 0 : 1;
-      $("input[name=build-time-toggle][value=" + newUseSubmissionDate + "]")
-        .prop("checked", true)
+      var newUseSubmissionDate = $("input[name=build-time-toggle]")
+        .is(":checked") ? 0 : 1;
+      $("input[name=build-time-toggle]")
+        .prop("checked", newUseSubmissionDate != 0 ? true : false)
```

Comment:
Now that we're using `checked` directly, I think we can simplify this toggling behaviour to look something like:

```js
let buildTimeToggle = $("input[name=build-time-toggle]");
buildTimeToggle.prop("checked", !buildTimeToggle.prop("checked"));
```